deploy/helm/kubecf/templates/brain-tests.yaml

```diff
@@ -221,4 +221,26 @@ roleRef:
   apiGroup: "rbac.authorization.k8s.io"
   name: "{{ .Release.Namespace }}-{{ .Release.Name }}-test-cluster-role"
 
+# Cluster role binding for service account "tests-brain" and cluster role "cluster-admin"
```

Comment:
We should describe _why_ we need this cluster role binding (as in, the specifics of the tests that require it, not just "for brains 🧟"

Reply:
See commit 8c3639e66e3f7b076403e5bfcbac35c26769fdd1 which has a larger comment in the values.yaml ... Is that one better ? If yes, I can copy the same text over to here.

Reply:
Yes, it is much better :)

Reply:
Ok. Commit copying it coming up

Reply:
Commit aa83f057d613103cdae30cacc3b036784505d591